android/src/main/java/com/github/mustafaozhan/ccc/android/ui/main/MainActivity.kt

```diff
@@ -31,13 +31,20 @@ class MainActivity : BaseActivity() {
         super.onCreate(savedInstanceState)
         Logger.i { "MainActivity onCreate" }
         installSplashScreen()
-        AppCompatDelegate.setDefaultNightMode(mainViewModel.getAppTheme())
+        AppCompatDelegate.setDefaultNightMode(mapAppTheme())
         setContentView(R.layout.activity_main)
         checkDestination()
         observeEffects()
         mainViewModel.checkReview()
     }
 
+    private fun mapAppTheme(): Int = when (mainViewModel.getAppTheme()) {
+        -1 -> AppCompatDelegate.MODE_NIGHT_NO
```

Comment:
Let's not use magic numbers, you can create a method by taking advantage of `AppTheme.getThemeByValue(appThemeValue)`
Something like this
```kotlin
fun updateAppTheme(appThemeValue: Int) = AppCompatDelegate.setDefaultNightMode(
    when (AppTheme.getThemeByValue(appThemeValue)) {
        AppTheme.LIGHT -> AppCompatDelegate.MODE_NIGHT_NO
        AppTheme.DARK -> AppCompatDelegate.MODE_NIGHT_YES
        else -> AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM
    }
)
```
and let's put this into `Localization.kt` and rename it to something more general like `DeviceUtil.kt`

and use this method in `MainActivity.kt` and `SettingsFragment.kt` so that we will get rid of `AppCompatDelegate` imports in the views 🙂